### ISSUE TITLE
Dashboard should display only confirmed cases

### ISSUE THREAD

**Issue by faatihi** [hzi-braunschweig/SORMAS-Project#1869]
### Feature Description
The dashboard is meant to give a quick overview of important information. Cases that are not confirmed are not that important to be included in the default content of the dashboard.

![image](https://user-images.githubusercontent.com/1176692/79746850-31f81c00-82fa-11ea-95fb-86d4bc61e0cb.png)

The above section, however, should remain as it is. That is, display all cases since it further breaks them down by classification. Other sections should only focus on confirmed cases in order to remove the noise from unconfirmed cases.

### Problem Description
Currently, users go through lots of customizations to get the dashboard to present them with the data they want to see. And that's every time they visit the page.

**Comment by Iheanacho2027** [comment]
@Chinedar @foyiri

**Comment by bernardsilenou** [comment]
I am hesitant to agree with this point of view. At the end of a cases' life, it should either be classified as confirmed (based on lab sample results) or not a case (if we have a  true negative lab test result). Suspected is just a state and our goal should be to limit the number of suspected cases by testing them.

If the dashboard has much suspected cases, it is a drive for us to test more of them or it is just telling us that out case definition is very week.

**Comment by Chinedar** [comment]
I agree with Bernard on this. All classifications are a reflection of
progress and outbreak cycle in general. We cannot have the dashboard only
show confirmed cases, we can however have a dynamic dashboard that affords
viewers the option to limit choice of view (based on the various
classifications).

**Comment by faatihi** [comment]
I understand that the dashboard needs to show progress on classification of cases. I think the statistics component (right below the slideshow) does a great job giving us insight into what's happening. That alone, in my opinion, is enough. The rest of the dashboard then focuses on confirmed cases only.
